package.json

```diff
@@ -194,7 +196,7 @@
     "passport-jwt": "^4.0.0",
     "passport-local": "^1.0.0",
     "passport-magic": "^1.0.0",
-    "pg": "^8.0.3",
+    "pg": "^8.7.1",
```

Comment:
Any other changes we should know about re: this pg bump?

Reply:
I may have run `yarn add pg` when I was working with Socket.io which upgraded the current `pg` version. It is only used for Socket.io and dbNode and both seem to be working fine with this updated version. Don't think it's an issue and isn't related to any other changes.